lib/mapObjects/CObjectClassesHandler.h

```diff
@@ -172,6 +190,10 @@ class DLL_LINKAGE AObjectTypeHandler : public boost::noncopyable
 			h & typeName;
 			h & subTypeName;
 		}
+		if(version >= 778)
```

Comment:
The problem is that for any old save nearly all sounds will not work. You removed all hardcoded sounds, but do not provide defaults for old saves.

Reply:
That's true, but there no easy way to partially reload JSON...